components/metadata.js

```diff
@@ -47,4 +43,29 @@ module.exports = async function getMetadata(argv, cli) {
     metadata,
     checker
   };
+}
+
+module.exports = async function getMetadata(argv, cli) {
+  const credentials = await auth({
+    github: true
+  });
+  const request = new Request(credentials);
+
+  if (!argv.assignee) {
+    return getOneMetadata(argv, cli, request);
+  } else {
+    // get all
+    // Example: repo:nodejs/node is:pr is:open assignee:srl295
+    const myQuery = new PRQuery(argv, cli, request);
+    const prids = await myQuery.getPRs();
+    for (const prid of prids) {
+      cli.separator(`${prid}`);
+      // new argv, with the prid
+      const nargv = Object.assign({}, argv, { prid });
+      delete nargv.assignee;
+      await getOneMetadata(nargv, cli, request);
+      cli.separator();
+    }
+    return {}; // ?
```

Comment:
not sure what the return should be here

Reply:
I think a better way to handle multiple requests is to add a different method for it, which calls the old `getMetadata`, and then call it from the command handler, instead of overwriting the method that's used to querying single requests. That also helps you avoid the `nargv.assignee` tinkering. In addition you may want to update the `fs.writeFileSync` call in `getMetadata` to `fs.appendFile` instead.

https://github.com/nodejs/node-core-utils/blob/1e9daeb15475efdfea4a418686395299ad892149/components/git/metadata.js#L83-L88